### DISCUSSION TITLE
Allow collapsible in chart widget?

### DISCUSSION THREAD

**Discussion by wotta** [filamentphp/filament#10036] [Ideas]
After a brief discussion with another member on [X](https://twitter.com/wottavm/status/1731283868454465551) on the response of the filament core team, I would like to pitch the idea of adding collapsable logic to the `chart-widget.blade.php`.

My main reason for the request is that the chart widget takes in quite some space on the list pages for a resource. See below.
![image](https://github.com/filamentphp/filament/assets/13332240/fc03f81f-36e7-4181-82f6-1af70f6cff14)

Adding the collapsible option makes it more manageable, especially when combined with the option to keep it auto-collapsed.
![image](https://github.com/filamentphp/filament/assets/13332240/e50489d7-c66f-48ac-bef9-8afe7a835935)

**Comment by SupianIDz** [comment]
@wotta  I hope my [PR](https://github.com/filamentphp/filament/pull/14992 ) gets merged so the feature you want is available.
